Resources/PO/UserSettings.robot

```diff
@@ -0,0 +1,21 @@
+*** Settings ***
+Library  SeleniumLibrary
+Library  String
+
+
+*** Variables ***
+${USERNAME_LOCATOR} =  //input[@id="profile_form_username"]
+${USER_SETTINGS_SAVE_LOCATOR} =  //input[@value="Save"]
+${USER_PROFILE_SUCCESS_ALERT} =  //div[@class="panel-body"]
+
+
+*** Keywords ***
+Input New Username And Save
+    Wait Until Page Contains Element  ${USERNAME_LOCATOR}
+    ${random_string} =  String.Generate Random String
+    Set Global Variable  ${NEW_USERNAME}  ${random_string}
```

Comment:
try to avoid using Global / test / suite variables in keywords

it's pretty difficult to found and control content of it